Makefile

```diff
@@ -0,0 +1,20 @@
+REG ?= quay.io
+ORG ?= kiagnose
+CHECKUP_IMAGE_NAME ?= kubevirt-dpdk-checkup
+CHECKUP_IMAGE_TAG ?= latest
+GO_IMAGE_NAME := docker.io/library/golang
+GO_IMAGE_TAG := 1.19
+BIN_DIR = $(CURDIR)/_output/bin
+CRI_BIN ?= $(shell hack/detect_cri.sh)
+
+build:
+	$(CRI_BIN) run --rm \
+	           --volume `pwd`:$(CURDIR):Z \
+	           --workdir $(CURDIR) \
+	           --user $(shell id -u):$(shell id -g) \
+	           -e XDG_CACHE_HOME=/tmp/.cache \
```

Comment:
Why are these necessary?

Reply:
running with docker results with the output folder having root permissions.
In order to fix that I added a specific --user parameters that levels the permissions to the one running the build.
But by doing that I ran into a different problem - docker build is creating a cache folder on `/` folder and that is forbidden for a non-root user. the fix is to set `XDG_CACHE_HOME` to a path that does not require root write permissions.
I looks a bit messy but I think it's worth the trouble in order to have the output folder non-root

Reply:
Added [here](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/6fb2fa3018a782b5a5c376eb66b55743d9f440bb..ed3f3d9cdec38970c06d397cbd8c528b858e9c88)